### PR TITLE
fix: root route for uploads only & remove swift refs

### PR DESCRIPTION
**Pull request by TacticalTechJay**
- readd root route for uploads only
 - catch 1 edge case for root route (/dashboard)

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # every field in here is optional except, CORE_SECRET and CORE_DATABASE_URL.
 # if CORE_SECRET is still "changethis" then zipline will exit and tell you to change it.
 
-# if using s3/swift make sure to comment out the other datasources
+# if using s3 make sure to comment out the other datasources
```
Comment (axisiscool) [inline .env.local.example]:
```suggestion
# if using s3/supabase make sure to comment out the other datasources
```
Replace references

File: .env.local.example [hunk 2]
```diff
@@ -25,15 +25,6 @@ DATASOURCE_S3_REGION=us-west-2
...
 
-# or you can use swift
-DATASOURCE_TYPE=swift
-DATASOURCE_SWIFT_CONTAINER=container
-DATASOURCE_SWIFT_AUTH_ENDPOINT="https://something/v3"
-DATASOURCE_SWIFT_USERNAME=username
-DATASOURCE_SWIFT_PASSWORD=password
-DATASOURCE_SWIFT_PROJECT_ID=project_id
-DATASOURCE_SWIFT_DOMAIN_ID=domain_id
-
```
Comment (axisiscool) [inline .env.local.example]: Replace this with subapase env keys and values

File: src/lib/config/validateConfig.ts [hunk 3]
```diff
@@ -233,21 +221,6 @@ export default function validate(config): Config {
...
-          errors.push('datasource.swift.project_id is a required field');
-        if (!validated.datasource.swift.auth_endpoint)
-          errors.push('datasource.swift.auth_endpoint is a required field');
-        if (!validated.datasource.swift.password)
-          errors.push('datasource.swift.password is a required field');
-        if (!validated.datasource.swift.username)
-          errors.push('datasource.swift.username is a required field');
-        if (errors.length) throw { errors };
-        break;
-      }
```
Comment (axisiscool) [inline src/lib/config/validateConfig.ts:250]: Validate supabase config instead

**Review by axisiscool — CHANGES_REQUESTED** [review]

**Review by diced — APPROVED** [review]

**Review by axisiscool — APPROVED** [review]
william zhou approves

**Comment by diced** [comment]
this will fix #214 btw

**Pull request merged**